### PR TITLE
Fix allocation size

### PR DESCRIPTION
**Pull request by Jayashsatolia403**
### Changes
-

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber:
- 0chain:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-june-5  |
| gosdk  | sprint-june-5  |
| zboxcli  | sprint-june-5  |
| zwalletcli  | sprint-june-5  |
| 0chain  | sprint-june-5  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | not_available  |
| gosdk  | sprint-june-5  |
| zboxcli  | not_available  |
| zwalletcli  | sprint-june-5  |
| 0chain  | not_avaliable  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-june-5  |
| gosdk  | fix/alloc-size  |
| zboxcli  | sprint-june-5  |
| zwalletcli  | sprint-june-5  |
| 0chain  | fix/size  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-june-5  |
| gosdk  | sprint-june-5  |
| zboxcli  | fix/alloc-size  |
| zwalletcli  | sprint-june-5  |
| 0chain  | fix/size  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-june-5  |
| gosdk  | fix/alloc-size  |
| zboxcli  | fix/alloc-size  |
| zwalletcli  | sprint-june-5  |
| 0chain  | fix/size  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-june-5  |
| gosdk  | fix/alloc-size  |
| zboxcli  | fix/alloc-size  |
| zwalletcli  | sprint-june-5  |
| 0chain  | fix/size  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-june-5  |
| gosdk  | sprint-june-5  |
| zboxcli  | sprint-june-5  |
| zwalletcli  | sprint-june-5  |
| 0chain  | fix/size  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

File: zboxcore/sdk/chunked_upload.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func CreateChunkedUpload(
 	}
 
 	opCode := OpUpload
-	spaceLeft := allocationObj.Size
+	spaceLeft := int64(float64(allocationObj.Size) * float64(allocationObj.DataShards+allocationObj.ParityShards) / float64(allocationObj.DataShards))
```
Comment (dabasov) [inline zboxcore/sdk/chunked_upload.go:118]: why should we adjust it?

**Review by dabasov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**